internal/object/test/object.service_test.go

```diff
@@ -28,4 +36,137 @@ func (t *ObjectServiceTest) SetupTest() {
 		BucketName: "mock-bucket",
 		Endpoint:   "mock-endpoint",
 	}
+	t.uploadObjectRequest = &proto.UploadObjectRequest{
+		Filename: "object",
+		Data:     []byte("data"),
+	}
+}
+
+func (t *ObjectServiceTest) TestUploadInternalError() {
+	repo := mock_object.NewMockRepository(t.controller)
+	service := object.NewService(repo, t.conf, t.logger, utils.NewRandomUtils())
+
+	repo.EXPECT().Upload(t.uploadObjectRequest.Data, t.conf.BucketName, gomock.Any()).Return("", "", fmt.Errorf("error"))
+
+	_, err := service.Upload(context.Background(), t.uploadObjectRequest)
+
+	t.EqualError(err, "rpc error: code = Internal desc = Internal server error")
+}
```

Comment:
This is my recommended test function (if there's better way to implement it, feel free to discuss it)
```go
func (t *ObjectServiceTest) TestUploadInternalError() {
	repo := mock_object.NewMockRepository(t.controller)
	repo.EXPECT().Upload(t.uploadObjectRequest.Data, t.conf.BucketName, gomock.Any()).Return("", "", fmt.Errorf("error"))

	svc := object.NewService(repo, t.conf, t.logger, utils.NewRandomUtils())

	expectedErr := status.Error(codes.Internal, constant.InternalServerErrorMessage).Error()

	actual, err := svc.Upload(context.Background(), t.uploadObjectRequest)

	t.Nil(actual)
	t.EqualError(err, expectedErr)
}
```
- instead of a hard code error message, you can use `status.Error(codes.Internal, constant.InternalServerErrorMessage).Error()` to create error (see it in `/constant/error.constant.go`
- `_, err := svc.Upload(...)` should be changed to `actual, err := svc.Upload(...)` and you have to assert the `actual` should be `nil` if an error case happens